packages/common/services/console-logger.service.ts

```diff
@@ -185,23 +185,27 @@ export class ConsoleLogger implements LoggerService {
   ) {
     const color = this.getColorByLogLevel(logLevel);
```

Comment:
```suggestion
    const color = this.getColorByLogLevel(logLevel);
```
This line doesn't seem to be used anymore